### PR TITLE
Enable aquavanjaram on staging

### PR DESCRIPTION
**Pull request by vin-huang**
1. fundemtanl for gfx940/941/942
2. Logic YAML for hipblaslt
3. XF32 support for hipblaslt and tensilelte
4. Sparse MFMA support

### CODE REVIEW

**Comment by jichangjichang** [comment]
These files needs clang-reformat.
clients/include/hipblaslt_test.hpp
clients/include/testing_matmul.hpp
clients/include/type_dispatch.hpp
clients/include/utility.hpp
library/src/amd_detail/rocblaslt/include/rocblaslt-auxiliary.h
library/src/amd_detail/rocblaslt/include/rocblaslt-types.h
library/src/amd_detail/rocblaslt/src/include/tensile_host.hpp
library/src/amd_detail/rocblaslt/src/rocblaslt_auxiliary.cpp
library/src/amd_detail/rocblaslt/src/rocblaslt_mat.hpp
library/src/amd_detail/rocblaslt/src/tensile_host.cpp
tensilelite/Tensile/Source/client/source/DataInitialization.cpp
tensilelite/Tensile/Source/client/source/ReferenceValidator.cpp
tensilelite/Tensile/Source/lib/include/Tensile/ContractionProblem.hpp
tensilelite/Tensile/Source/lib/include/Tensile/DecisionTree.hpp
tensilelite/Tensile/Source/lib/include/Tensile/DistinctType.hpp
tensilelite/Tensile/Source/lib/include/Tensile/KernelArguments.hpp
tensilelite/Tensile/Source/lib/include/Tensile/TensorDescriptor.hpp
tensilelite/Tensile/Source/lib/source/ContractionProblem.cpp
tensilelite/Tensile/Source/lib/source/ContractionSolution.cpp

**Comment by vin-huang** [comment]
clang-format, done.
please see the last commit

File: tensilelite/Tensile/Activation.py [hunk 1]
```diff
@@ -457,15 +457,18 @@ def getExpModule(self, cDataType, vgprIn, vgprOut):
                                        sdwa=SDWAModifiers(dst_sel=select_bit, dst_unused=UnusedBit.UNUSED_PRESERVE, \
                                                           src0_sel=select_bit), \
                                        comment="exp step 2"))
-                    module.add(SNop(waitState=0, comment="1 wait states")) #workaround for emulator
+                    if globalParameters["ArchCaps"][globalParameters["CurrentISA"]]["TransOpWait"]:
+                        module.add(SNop(waitState=0, comment="1 wait states"))
```
Comment (jichangjichang) [inline tensilelite/Tensile/Activation.py:461]: Please init the Caps in _initArchCaps() rather then check ISA version in kernel writer stage.
def _initArchCaps(isaVersion) -> dict:

Comment (vin-huang) [reply tensilelite/Tensile/Activation.py:461]: done

**Pull request merged**